Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -367,6 +367,24 @@
             "required": false,
             "visibleRule": "command = pack"
         },
+        {
```

Comment:
Can you update the minor version of the task to 152 in this file and drop the patch back to 0?  I believe it is currently 151.2 or similar and without a version bump it isn't going to deploy since it is on the latest of the last milestone still (during this next sprint it can stay at 152.0 until it branches into release again)

Reply:
Done